geode-management/src/main/java/org/apache/geode/cache/configuration/RegionType.java

```diff
@@ -47,5 +47,25 @@
 
   // this is used to represent regions not supported by the management V2 API. For example Gfsh can
   // create regions with "LOCAL*" types
-  UNSUPPORTED
+  UNSUPPORTED;
+
+  public boolean isProxy() {
```

Comment:
I like these new "is" methods but add javadocs to make clear that it returns true if the type is any of the types that are proxies/partitioned/replicate.
In the past on the core apis we had some confusion on methods like these since a method like isReplicate could mean "it is the REPLICATE type" instead of "it is one of the replication types".

Reply:
remove these methods for now, we don't need them. And we might change these types anyway.